test/activerecord_spanner_interleaved_table_7_1/models/album.rb

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Google LLC
```

Comment:
does `activerecord_spanner_interleaved_table_7_1` mean `< 7.1` or `>=7.1` or `==7.1`?

Reply:
It means `>=7.1`. I changed the name to better reflect that.